app/src/js/components/Collections/list.js

```diff
@@ -103,7 +98,10 @@ class CollectionList extends React.Component {
         </section>
         <section className='page__section'>
           <div className='heading__wrapper--border'>
-            <h2 className='heading--medium heading--shared-content with-description'>{strings.all_collections} <span className='num--title'>{count ? ` ${tally(count)}` : 0}</span></h2>
+            <h2 className='heading--medium heading--shared-content with-description'>
+              {hasTimeFilter ? strings.active_collections : strings.all_collections}
+              <span className='num--title'>{count ? `${tally(count)}` : 0}</span>
```

Comment:
Now that the leading space is removed, there should be no need for a template string.